examples/sendgrid-event-webhook-consumer/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM microsoft/dotnet:2.1-sdk AS build
+WORKDIR /App
+
+# copy csproj and restore as distinct layers
+COPY *.sln .
+COPY Src/EventWebhook/*.csproj ./Src/EventWebhook/
+COPY Tests/EventWebhook.Tests/*.csproj ./Tests/EventWebhook.Tests/
```

Comment:
Can this just copy the whole dir instead of multiple copy commands?

Reply:
@childish-sambino This help us leverage the docker's intermediate layer caching while building the image.